app/src/test/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenterTest.java

```diff
@@ -0,0 +1,63 @@
+package io.intrepid.contest.screens.contestcreation.categorieslist;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.intrepid.contest.models.Category;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.Mockito.verify;
+
+public class CategoriesListPresenterTest {
+    @Mock
+    CategoriesContract.View mockView;
+    private CategoriesContract.Presenter categoriesListPresenter;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+        categoriesListPresenter = new CategoriesListPresenter(mockView,
+                                                              TestPresenterConfiguration.createTestConfiguration());
+    }
+
+    @Test
+    public void presenterCanBeCreated() {
+        assertTrue(categoriesListPresenter != null);
+    }
+
+    @Test
+    public void displayContestWithoutCategoriesShowsExampleCategory() {
```

Comment:
Talk to Gabi about how she names her tests. We generally want them to be <funcation_name>should<what_it_does>when<conditions>. So this would be displayCategoriesShouldShowDefaultCategory. (no when needed in this condition)